lib/services/ngsi/entities-NGSI-LD.js

```diff
@@ -403,6 +403,70 @@ function addLinkedEntities(typeInformation, json) {
     });
```

Comment:
CHANGES_NEXT_RELEASE entry is missing in this PR.

Reply:
Fixed 8f7e8fa - The PR was split in two and a second CNR was not added.